src/docs/config.md

```diff
@@ -28,7 +28,7 @@
 | Property | Required | Default | Description |
 | :--- | :--- | :--- | :--- |
 | `revision` | | | TODO: maybe the user doesn’t need to know about this? |
-| `win_cmd` | ![check](/images/icons/checkmark.svg) | | The command to execute to spawn the “back-end” process. |
+| `win_cmd` | | | The command to execute to spawn the “back-end” process, leave this out use front end only |
```

Comment:
~leave this out use front end only~ This value may be omitted for front end only applications